src/coreclr/nativeaot/Runtime/eventpipe/ep-rt-aot.cpp

```diff
@@ -0,0 +1,502 @@
+#include <eventpipe/ep-rt-config.h>
```

Comment:
Add copyright header to all new files.

Reply:
Created https://github.com/dotnet/runtime/pull/82232 for this, thanks!